index.html

```diff
@@ -0,0 +1,879 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    <link rel="stylesheet" href="./css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+      integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Arbutus+Slab&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="shortcut icon" href="images/favicon.ico"/>
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav class="nav-bar">
+        <div class="icons">
+          <a href="#"><i class="fa fa-bars" aria-hidden="true"></i></a>
+          <a href="#"><i class="fa fa-search" aria-hidden="true"></i></a>
+        </div>
+        <div class="logo">
+          <img src="./images/logo.png" alt="" />
```

Comment:
- [ ] You are importing the logo with the wrong path, and it is not showing. The image in the project folder has a different extension (the PNG is in upper case)